src/functions.ts

```diff
@@ -17,12 +17,7 @@
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import * as express from 'express';
-import {CloudEventsContext} from './cloud_event_types/cloud_events_context';
-
-/**
- * @public
- */
-export {CloudEventsContext};
```

Comment:
this line was necessary to let developers import the `CloudEvent` interface from the functions framework. You renamed it `CloudEvent` so:

```typescript
export {CloudEvent};
```

Side note: I am surprised api-extractor did catch this.